scalableminds/webknossos
# Dataset Upload: Improve Progress Display

- While waiting for the finishUpload request (unpacks zip etc), don’t show 99.9% anymore, but instead 100% again, with added spinner and label »Processing uploaded files …«. I think the stalling at 99.9% for long-running finishUpload requests was confusing some users.
- Move the success toast below the sleep in the control flow, so that users do not wonder why there is another pause after “success”

Do you agree that these are sensible changes?

### URL of deployed dev instance (used for testing):
- https://uploadfinish.webknossos.xyz

### Steps to test:
- Upload a dataset (e.g. a few 1GB wkw zip like l4_sample)
- Upload page should not make you more impatient than necessary, specifically towards the end

### Issues:
- fixes #5615

------
